hack/tests/e2e-upgrade.sh

```diff
@@ -35,6 +35,7 @@ cp -a "$T/src/github.com/openshift/openshift-azure/_data" .
 
 set_build_images
 
+export AZURE_REGIONS=$(awk '/^location:/ { print $2 }' <_data/containerservice.yaml)
```

Comment:
I do not think this change should be here.  Why is it here?

Reply:
This change is here because there is currently a bug in `hack/tests/e2e-upgrade.sh` where a cluster could be created in one region and the upgrade is done with another region. See [these logs](https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_openshift-azure/1950/pull-ci-azure-master-upgrade-v7.0/361/build-log.txt) for an example of this bug in action. The 7.0 cluster is created in westeurope and then the upgrade to the latest plugin version is attempted in australiasoutheast :man_shrugging: